src/sentry/static/sentry/app/components/queryCount.tsx

```diff
@@ -20,37 +17,25 @@ type Props = {
  * Render nothing by default if `count` is falsy.
  */
 
-const QueryCount = ({
-  className,
-  count,
-  max,
-  hideIfEmpty = true,
-  inline = true,
-  hideParens = false,
-}: Props) => {
+const QueryCount = ({count, max, hideIfEmpty = true, hideParens = false}: Props) => {
   const countOrMax = defined(count) && defined(max) && count >= max ? `${max}+` : count;
-  const cx = classNames('query-count', className, {
-    inline,
-  });
 
   if (hideIfEmpty && !count) {
     return null;
   }
 
   return (
-    <div className={cx}>
+    <span className="query-count">
```

Comment:
Why still have the class if you deleted the styles?

Reply:
i think i was worried about someone using it as a selector, i can get rid of it